app/support/reservations/moving_up.rb

```diff
@@ -8,11 +8,12 @@ module Reservations::MovingUp
   # if there is no such time slot. For read-only purposes.
   def earliest_possible
     after = 1.minute.from_now
-    next_res = product.next_available_reservation(after,
-                                                  duration_mins.minutes,
-                                                  exclude: self,
-                                                  user: user,
-                                                  until: reserve_start_at)
+    next_res = product.next_available_reservation(after: after,
```

Comment:
I like the keyword arguments, but now that we're using them, how many `options` are there? Would it be clean enough to make them all top level arguments rather than having an `options` argument?